src/app/frontend/common/components/list/column/menu/component.ts

```diff
@@ -57,16 +55,8 @@ export class MenuComponent implements ActionColumn, OnDestroy {
     this.typeMeta = typeMeta;
   }
 
-  ngOnDestroy(): void {
```

Comment:
This Angular component requires an `ngOnDestroy()` method. 

Running `npm start` on this branch causes the following error:
```
[1] ERROR in src/app/frontend/common/components/list/column/menu/component.ts(42,14): error TS2420: Class 'MenuComponent' incorrectly implements interface 'OnDestroy'.
[1]   Property 'ngOnDestroy' is missing in type 'MenuComponent'.
[1]
[1] ℹ ｢wdm｣: Failed to compile.
```

Reply:
It's mainly because this class implements `OnDestroy` which requires this function. It should be removed from class definition.

Reply:
Yep, didn't notice as it was changing it multiple times. Thanks for checking that.